helpers/dbload.ts

```diff
@@ -96,6 +99,34 @@ UserLesson.belongsTo(User, {
 
 sequelize.sync({ alter: !!process.env.ALTER_DB })
 
+/* istanbul ignore next */
+if (process.env.NODE_ENV === 'development') {
```

Comment:
I choose this check over `process.env.NODE_ENV != 'production'` since it allows me to isolate dbload.test from grapql.test (which is too hard to mock properly, at least with my jest skills). For example, both Lesson and Challenge classes are extending sequelize Model, so when mocked both will share the same arguments calls. In some cases test won't even start due to circular references in files.